### ISSUE TITLE
How can i do sensors constant frequency

### ISSUE THREAD

**Issue by stiglioglu** [lgsvl/simulator#1652]
How do I constant frequency the frequencies of the sensors? For example, synchronized operation of lidar and camera at 5 Hz.

**Comment by heeen** [comment]
have you tried adding a matching frequency parameter to each

**Comment by stiglioglu** [comment]
yes i tried, they all work at 12hz but the returned data does not return synchronously. For example, if the camera data returns 100 data, 50 data is returned from the lidar.

**Comment by firatbozkaya** [comment]
Hi @stiglioglu, did you find a solution? I am having the same problem.
